skyportal/models.py

```diff
@@ -405,6 +405,8 @@ class Photometry(Base):
     altdata = sa.Column(JSONB)
     upload_id = sa.Column(sa.String, nullable=False,
                           default=lambda: str(uuid.uuid4()))
+    alert_id = sa.Column(sa.BigInteger, nullable=False, unique=True,
+                         default=lambda: np.random.randint(100, 9223372036854775807))
```

Comment:
Compute hash(mjd) and use it here instead of generating a random integer? A collision should practically never happen.

Reply:
@dmitryduev proposed solution  will break for multiple alerts off the same image

Reply:
ah I take it back, this is only for prv_candidates

Reply:
but wouldn't this break for multiple objects off the same exposure with PRV candidates?

Say I have object 1 on chip 3 of field 2
And object 2 on chip 6 of field 2 

Both have prv_candidates without alerts issued off the same prior exposure

mjd = 58000.

Both of these PRV candidates will map to the same alert ID under the proposed scheme